packages/@lwc/features/src/babel-plugin/index.js

```diff
@@ -38,6 +38,11 @@ module.exports = function({ types: t }) {
                     const didImportRuntimeFlags = specifiers.some(specifier => {
                         return specifier.local && specifier.local.name === RUNTIME_FLAGS_IDENTIFIER;
                     });
+                    if (didImportRuntimeFlags && !this.opts.prod) {
+                        throw new Error(
+                            `Invalid import of "${RUNTIME_FLAGS_IDENTIFIER}" from "${FEATURES_PACKAGE_NAME}". Runtime flags can only be imported by the compiler.`
```

Comment:
I don't understand the `Runtime flags can only be imported by the compiler.` part of the error message. Could you replace this section with details on how to fix this?